File: src/main/java/com/bokeunjeong/portfolio/repository/PortfolioContactRepository.java
package com.bokeunjeong.portfolio.repository;

import com.bokeunjeong.portfolio.model.Contact;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface PortfolioContactRepository extends JpaRepository<Contact, String> {

}

File: src/main/kotlin/com/bokeunjeong/portfolio/repository/PortfolioContactRepository.kt
package com.bokeunjeong.portfolio.repository

import com.bokeunjeong.portfolio.dto.PortfolioContactDto
import org.springframework.data.jpa.repository.JpaRepository

interface PortfolioContactRepository : JpaRepository<PortfolioContactDto, Int> {
    fun findByType(type: String): List<PortfolioContactDto>
}